velox/exec/tests/MultiFragmentTest.cpp

```diff
@@ -1783,5 +1780,61 @@ DEBUG_ONLY_TEST_F(MultiFragmentTest, exchangeStatsOnFailure) {
   ASSERT_TRUE(waitForTaskCompletion(producerTask.get(), 3'000'000));
 }
 
+TEST_F(MultiFragmentTest, earlyTaskFailure) {
+  setupSources(1, 10);
+  std::vector<std::shared_ptr<TempFilePath>> filePaths(
```

Comment:
Why not auto filePaths = filePaths_? Why do we need this copy?

Reply:
We don't need it. I copy over from the other test. Thanks!